src/spin1_beam_model/plotting.py

```diff
@@ -0,0 +1,144 @@
+import cmocean
```

Comment:
I don't think `cmocean` is listed anywhere as a formal dependency. We should either make this import optional or list it in the `setup.cfg`.

Reply:
It's in `extras_require`, in `plots`. I thought it seemed rather like an appendage, not core, so I thought it better there. What do you think?

Reply:
Ah okay. I didn't realize it's in the `setup.cfg` file. I think that's fine then.